containers/sandbox/Dockerfile

```diff
@@ -28,6 +28,11 @@ RUN mkdir -p -m0755 /var/run/sshd
 # symlink python3 to python
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
+# create and activate virtual environment
+ENV VIRTUAL_ENV=/app/.venv \
+    PATH="$VIRTUAL_ENV/.venv/bin:$PATH"
+RUN python3 -m venv $VIRTUAL_ENV
```

Comment:
It actually won't have any effect on the actual `python` being used? Since when the sandbox starts, the default python will still be `/usr/bin/python`.

<img width="1214" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/38853559/d3e34019-d640-44fd-8530-2929028095f7">

You can reproduce this by locally build this image and run with the built sandbox image (setting `sandbox_container_image`):

`python3 opendevin/runtime/docker
/ssh_box.py`

Reply:
Btw, I just realized -- I think these two lines are still not used?

<img width="660" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/38853559/b7f9e15c-2404-435b-8d20-7b6fd9b81f8e">

Reply:
That is, the package you installed is still uses the system wide python? -- I think if you do environment activate on `Dockerfile`, it probably doesn't work when the container is running. Nonetheless I think the testcase is super helpful!
<img width="567" alt="image" src="https://github.com/OpenDevin/OpenDevin/assets/38853559/d51b2df7-0718-44d5-829d-5ae8559d2e78">

Reply:
> That is, the package you installed is still uses the system wide python? -- I think if you do environment activate on `Dockerfile`, it probably doesn't work when the container is running. Nonetheless I think the testcase is super helpful!

Yes, you are right. And I found a better solution from the community that resolves the issue of installation, so I closed the PR. Thank you for your patient guidance.